### PR TITLE
Fix enterprise contract violations

### PR DESCRIPTION
**Pull request by sayan-biswas**
Changes:
- Required labels added
- Hermetic build enabled
- Source image build enabled

### CODE REVIEW

**Comment by sayan-biswas** [comment]
/hold

The only EC violation remaining is the CVE blockers, which I'll fix in another PR before merging this.

**Review by avinal — APPROVED** [review]
/lgtm

**Pull request merged**
